proposals/002_Summary.md

```diff
@@ -0,0 +1,111 @@
+# Collection Summary
+
+## Metadata
+
+| Tag | Value |
+| ---- | ---------------- |
+| Proposal | [Summary](https://github.com/tschaub/ogcapi-features/tree/summary/proposals/002_Summary.md) |
+| Authors | Various |
+| Review Manager | TBD |
+| Status | Draft |
+| Implementations | None |
+| Issues | None |
+| Previous Revisions | None |
+
+## Introduction
+
+This proposal suggests that the spec define a `summary` resource (all names TBD) that describes a collection.  It may be that this information belongs in the existing collection info (e.g. at `/collections/{id}`), but for the purposes of this doc, we'll say it is going to be a new resource (e.g. at `/collections/{id}/summary`).
+
+## Motivation
+
+There are few motivations for this:
+
+ * A client should have a way to get the formal schema for items in a collection if one exists.  The current suggestion to use a link with `rel="describedBy"` doesn't fully address this need since it doesn't provide a robust way for a client to get the schema for multiple content types (for example, if a service supports GML, KML, and GeoJSON output, the `type` and `rel` of a link is not enough information for a client to know which schema applies to which output format).
+
+ * A "filter" extension should have a way to describe which properties are filterable and how clients should access a specific property in a filter expression.
+
+ * A "search" extension should have a way to describe which properties can be used in sorting.
+
+ * An "editing" extension should have a way to describe which properties are writeable (e.g. in an update) or might want to allow for a different schema for each method (e.g. one schema for create, one for update).
+
+## Proposed solution
+
+## Example
+
+Here is an example of what this summary resource might look like for a "buildings" collection (e.g. at https://example.com/collections/buildings/summary)
+
+```json
+{
+  "properties": {
+    "application/geo+json": {
+      "height": {
+        "schema": {
+          "href": "http://example.com/collections/buildings/schema#/definitions/height"
+        }
+      },
+      "age": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
+        },
+        "filter": {
+          "disallowed": true
+        },
+        "search": {
+          "unsortable": true
+        }
+      },
+      "taxID": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
```

Comment:
```suggestion
          "href": "http://example.com/collections/foo/schema#/definitions/taxId"
```

Reply:
I changed the property name to `taxId` (lowercase `d`) in 5a0f9d83042d06af9bf46d29321d2a4fdcf123c7.